lldb/test/API/lang/swift/printdecl/TestSwiftTypeLookup.py

```diff
@@ -56,6 +56,12 @@ def test_swift_type_lookup(self):
                 'func bar()',
                 'var b'])
 
+        # Regression test. Ensure "<could not resolve type>" is not output.
+        self.expect(
+            "type lookup String",
+            matching=False,
+            substrs=["<could not resolve type>"])
```

Comment:
Assuming that the "not" in the comment is correct — should we also/just add a positive test for the expected result?

Reply:
there's a positive test a few lines above. They use substring checks, which allows them to currently pass despite the output containing `<could not resolve type>`. Maybe you know a technique, but it looks to me that the positive test would have to be expanded to match the entire `type lookup` output. That would be a bit noisy, and possibly makes it more fragile in the face of changes.

What I have found myself wanting is for the `expect` function to have a second parameter that allows you to express which strings are not present, in addition to which strings are present. Ex:

```python
        self.expect(
             "type lookup String",
             substrs=["struct String {"],
             not_substrs=["<could not resolve type>"])
```

Reply:
> What I have found myself wanting is for the expect function to have a second parameter that allows you to express which strings are not present, in addition to which strings are present. Ex:

That sounds like a reasonable patch for llvm.org!

Reply:
Yes I might do that. What should I do with this test in the mean time?

Reply:
We can leave it as is. Don't let the perfect be the enemy of the good.